data/json/items/gun/762.json

```
@@ -115,5 +115,80 @@
     ],
     "magazines": [ [ "762", [ "762x39_clip" ] ] ],
     "flags": [ "RELOAD_ONE", "NEVER_JAMS" ]
+  },
+  {
+    "id": "wasr10",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "reload_noise_volume": 10,
+    "name": { "str": "WASR-10" },
+    "description": "A semi-automatic-only civilian-legal Kalashnikov-pattern rifle, imported from Romania.  A valuable survivors' commodity.",
+    "weight": "3300 g",
+    "volume": "1750 ml",
+    "price": 79500,
+    "to_hit": -1,
+    "bashing": 12,
+    "material": [ "steel", "wood" ],
+    "symbol": "(",
+    "color": "brown",
+    "ammo": "762",
+    "skill": "rifle",
+    "dispersion": 180,
+    "durability": 8,
+    "modes": [ [ "DEFAULT", "semi-auto", 1 ] ],
+    "barrel_length": "250 ml",
+    "valid_mod_locations": [
+      [ "accessories", 4 ],
+      [ "barrel", 1 ],
+      [ "bore", 1 ],
+      [ "brass catcher", 1 ],
+      [ "grip", 1 ],
+      [ "mechanism", 4 ],
+      [ "muzzle", 1 ],
+      [ "rail", 1 ],
+      [ "sights", 1 ],
+      [ "sling", 1 ],
+      [ "stock", 1 ],
+      [ "underbarrel", 1 ]
+    ],
+    "magazines": [ [ "762", [ "akmmag", "akmbigmag" ] ] ],
+    "flags": [ "NEVER_JAMS" ]
+  },
+  {
+    "id": "draco",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "reload_noise_volume": 10,
+    "name": { "str": "Mini Draco" },
+    "description": "Essentially a semi-auto AK with a short barrel and no stock, this unwieldy \"pistol\" has seen some popularity with civilian shooters and gangbangers alike.",
+    "weight": "2560 g",
+    "volume": "1250 ml",
```

```suggestion
    "volume": "2662 ml",
```
Mini Draco, overall length 17.50": https://www.centuryarms.com/mini-draco-pistol.html
Removing mag for binarization: https://imgur.com/a/as3lvUx
Surface area(square in =374.986 sq. cm): https://imgur.com/a/uZKEwEj
Using same width as WASR-10, unless you find a different source.

---

Using the width you obtained from homemade receiver
```suggestion
    "volume": "1286 ml",
```